frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/index.tsx

```diff
@@ -32,18 +41,26 @@ const DataFormat: FC<Props> = ({
         name="dataFormat"
         disabled={isDisabled}
         onChange={handleChange}
-        selectedValue={format}
+        selectedValue={selectedFormat}
       >
         <Radio
           disabled={!availableFormats.includes(DATASET_ASSET_FORMAT.H5AD)}
           label=".h5ad (AnnData v0.7)"
           value={DATASET_ASSET_FORMAT.H5AD}
         />
-        <Radio
-          disabled={!availableFormats.includes(DATASET_ASSET_FORMAT.RDS)}
-          label=".rds (Seurat v3)"
-          value={DATASET_ASSET_FORMAT.RDS}
-        />
+        <Tooltip
+          disabled={!rdsSkipped}
+          interactionKind={PopoverInteractionKind.HOVER}
+          content="An .rds (Seurat v3) download is unavailable due to limitations in the R dgCMatrix sparse matrix class."
```

Comment:
The `a` vs `an` acronym-phoneticization conflict that sometimes arises is a real pain. But I have to vote we go with what was in the ticket, per Bryan:

>Include hover text for this choice with a warning "A .rds (Seurat v3) download is unavailable due to limitations in the R dgCMatrix sparse matrix class."

Reply:
https://www.r-bloggers.com/2011/03/%E2%80%9Can-r-package%E2%80%9D-or-%E2%80%9Ca-r-package%E2%80%9D/

Reply:
Alrighty, Brian is overruled...